### PR TITLE
✨ Added ZDFmediathek presence

### PR DESCRIPTION
**Pull request by Reichmann-M**
# Proof of work

[www.zdf.de](https://www.zdf.de/) - Mainpage of Second German Television Streaming Portal
screen0

[ZDF Livestream](https://www.zdf.de/live-tv) - Second German Television Livestream
screen1

[ZDF Video-on-demand](https://www.zdf.de/dokumentation/zdfinfo-doku/im-land-der-feuerberge--island-100.html) - Normal show & movie streaming
screen2

### CODE REVIEW

File: websites/Z/ZDFmediathek/presence.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        : showTitleTag;
+      [, presenceData.endTimestamp] = timestamps;
+      presenceData.buttons = [
+        { label: (await strings).buttonWatchVideo, url: prevUrl }
+      ];
+      if (video.paused) {
+        presenceData.smallImageKey = "pause";
+        presenceData.smallImageText = (await strings).pause;
+        presenceData.startTimestamp = 0;
+        presenceData.endTimestamp = 0;
```
Comment (Bas950) [inline websites/Z/ZDFmediathek/presence.ts:117]: dont set to 0
do:
delete presenceData.startTimestamp

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/Z/ZDFmediathek/presence.ts (hunk 1 above)
Comment (Reichmann-M) [reply websites/Z/ZDFmediathek/presence.ts]: Should be have taken care of by latest commit.
Thanks for the advice 👍 
I'm also gonna change this in my last presence contribution (ARD Mediathek #3950) via PR.

File: websites/Z/ZDFmediathek/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "../../../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/"
+    }
+  }
```
Comment (ririxidev) [inline websites/Z/ZDFmediathek/tsconfig.json:6]: will be nice if you will fix spcaing

**Review by Bas950 — APPROVED** [review]
LGTM

**Review by rxri — APPROVED** [review]

**Pull request merged**